src/extension/background.js

```diff
@@ -80,9 +80,9 @@ function getConfigFromTabUrl(tabUrl) {
         };
       } else {
         // check if url is known in discovery cache
-        const discoveryCache = DISCOVERY_CACHE.find(({ url: cacheUrl }) => cacheUrl === tabUrl);
-        if (discoveryCache) {
-          const { owner, repo } = discoveryCache.results.find((r) => r.originalRepository) || {};
+        const discoveryCache = queryDiscoveryCache(tabUrl);
+        if (discoveryCache.length > 0) {
```

Comment:
this `if` is not really needed. since `find` will not find anything on an empty array